nativelib/src/main/resources/scala-native/gc/shared/Parsing.c

```diff
@@ -7,6 +7,10 @@
 #include <windows.h>
 #endif
 
+// Helpers for compile time memory
```

Comment:
For the sake of people reading alone perhaps something like
```
// Idiomatic 'stringify' Helper for compile time memory
```

Reply:
Yes, modified slightly.